src/ripple/nodestore/impl/DatabaseShardImp.cpp

```diff
@@ -1350,12 +1484,48 @@ DatabaseShardImp::getCache(std::uint32_t seq)
     return std::make_pair(pCache, nCache);
 }
 
-std::uint64_t
-DatabaseShardImp::available() const
+bool
+DatabaseShardImp::sufficientStorage(
+    std::uint32_t numShards,
+    bool isHistorical,
```

Comment:
Replacing the `bool` with an enum would make the calling sites clearer.

Reply:
Done